examples/resources/datadog_authn_mapping/resource.tf

```diff
@@ -0,0 +1,11 @@
+# Source the role
+data "datadog_role" "ro_role" {
+  filter = "Datadog Read Only Role"
+}
+
+# Create a new AuthN mapping
+resource "datadog_authn_mapping" "foo" {
```

Comment:
Since this is an example for the public docs, let's try to use realistic "good" naming, replacing "foo" with "dev_ro_role_mapping" or something more meaningful like that?